We use different paths for some files than Gentoo does.

diff --git a/src/localed.c b/src/localed.c
index 008ed16..1e53dae 100644
--- a/src/localed.c
+++ b/src/localed.c
@@ -1280,9 +1280,9 @@ localed_init (gboolean _read_only)
     struct xorg_confd_parser *x11_parser = NULL;
 
     read_only = _read_only;
-    kbd_model_map_file = g_file_new_for_path (PKGDATADIR "/kbd-model-map");
-    locale_file = g_file_new_for_path (SYSCONFDIR "/env.d/02locale");
-    keymaps_file = g_file_new_for_path (SYSCONFDIR "/conf.d/keymaps");
+    kbd_model_map_file = g_file_new_for_path (PKGDATADIR "/openrc-settingsd/kbd-model-map");
+    locale_file = g_file_new_for_path (SYSCONFDIR "/profile.d/locale");
+    keymaps_file = g_file_new_for_path (SYSCONFDIR "/conf.d/loadkeys");
 
     /* See http://www.gentoo.org/doc/en/xorg-config.xml */
     x11_gentoo_file = g_file_new_for_path (SYSCONFDIR "/X11/xorg.conf.d/30-keyboard.conf");
